vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -354,7 +354,11 @@ def __init__(
 
         self.cutlass_fp8_supported = cutlass_fp8_supported()
         self.flashinfer_moe_backend: FlashinferMoeBackend | None = None
-        if envs.VLLM_USE_FLASHINFER_MOE_FP8 and has_flashinfer_moe():
+        if (
+            envs.VLLM_USE_FLASHINFER_MOE_FP8
+            and has_flashinfer_moe()
+            and self.moe.is_act_and_mul
+        ):
```

Comment:
For NemotronH, `self.flashinfer_moe_backend` will end up being `None`. What implementation ends up getting used in this case?

Reply:
triton kernels. This is currently the only code path available with `is_act_and_mul=False`

Reply:
Agreed. We can follow-up on this discussion internally